assets/airlines/ldm.json

```diff
@@ -4,31 +4,21 @@
     "callsign": {
         "name": "Lauda Motion",
         "callsignFormats": [
-            "###",
-            "##"
+            "#@@",
+            "##@",
+            "##@@"
         ]
     },
     "fleets": {
         "default": [
-            ["A320", 5],
-            ["A321", 4],
-            ["B738", 6]
-        ],
-        "short": [
-            ["A320", 5],
-            ["A321", 4]
-        ],
-        "wetlease": [
```

Comment:
1. When removing a fleet from an airline file, please always check if there are any other airports that use this fleet (In this case you removed the b738 fleet which is used by EDDL and might cause errors when playing this airport and the sim wants to load a not existing fleet). Many editors have a "search in folder" function, search the airport folder for "ldm/b738" to see if there are any occurences.

2. The "b738" as well as the "wetlease" fleet should be kept here as laudamotion is a subsiduary of Ryanair Holdings and ryanair operates a couple of wetlease flights with 737s for LDM:
![Screenshot_253](https://user-images.githubusercontent.com/24833249/57111621-24d6cb80-6d3d-11e9-95d1-99e494904c05.png)

Reply:
Whoops, yeah I didn't think thru what could happen if I deleted something that is used somewhere else, but thanks for pointing that out in that files. (And I have no idea about the current situation with Ryanair and Laudamotion and I guess Lufthansa in some way too? Super confusing.)
~Done